### ISSUE TITLE
Missing DLLs in the bin folder

### ISSUE THREAD

**Issue by donovaly** [apeltauer/FreeCAD#18]
During the release process of FC 0.19.2 I found out that for older Windows PCs we need to deliver some more DLL files. Here is the collection of all DLLs I found so far: https://github.com/donovaly/FreeCADInstProj/releases/download/LibPack-fixes-1/Missing-DLLs-for-LibPack-12-5-3.zip

Can you please add them to the next LibPack?

**Comment by apeltauer** [comment]
Hi, what kind of 3rd party dlls are these? Can you give me  link to lib?

**Comment by donovaly** [comment]
I found that one needs for Python for debugging purposes the MKL DLLs: https://en.wikipedia.org/wiki/Math_Kernel_Library

So I did `pip install mkl` and got them in "C:\Program Files\Python38\Library\bin"

Despite I have an AMD processor, I needed the intel MKL and I needed to rename it.

Other files (like lapack) are needed by Python's SciPy, also to be able to debug.

I don't remember where I got the files from, I always searched "python xxx.dll" whenever I encountered a missing DLL and then often a stackoverflow thread pointed me to where I could get it.

**Comment by donovaly** [comment]
I have to correct myself - they are not only necessary for debug purposes. I remember I needed them while I was just running FC as a normal user on my old laboratory laptop. And this has an Intel i3 processor. Therefore it makes sense that I needed the Intel MKL Dll. I also remember that I needed liblapack.dll and the like.

When I released FC 0.19.2 I spent an epic night shift with 5 PCs around any by try and error I collected all DLLs I needed until I could perform an FEM simulation with FC.
Therefore I distribute the DLLs. So far we haven't get feedback about missing DLLs, so obviously I made a good catch ;-)
